packages/connectivity/package.json

```diff
@@ -31,7 +31,7 @@
     "test:unit": "jest",
     "coverage": "jest --coverage",
     "lint": "eslint --ext .ts . && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -c",
-    "lint:fix": "eslint --ext .ts . --fix && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w",
+    "lint:fix": "TIMING=1 eslint --ext .ts . --fix --quiet && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --loglevel error",
```

Comment:
So what does this TIMING=1 do?
Other than that LGTM

Reply:
@florian-richter could you enlighten us?

Reply:
This was a recommendation in the turbo docs to make linting output nicer. It sets an environment variable for the time of this command which causes eslint to show a table of the longest running eslint tasks. I found it helpful or at least unobtrusive so I kept it.